### PR TITLE
Create apachefoundation.md

### PR DESCRIPTION
**Pull request by reginankenchor**

### CODE REVIEW

**Review by RichardLitt — CHANGES_REQUESTED** [review]
I don't think the contact form needs to be here. Can you edit that?

File: organizations/apachefoundation.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Before reaching out by email or mailing, it is advisable to get familiar with the various contact options available at [Apache Contact Information](https://www.apache.org/foundation/contact).  
+For general inquiries, you can reach out to [info@apache.org](mailto:apache@apache.org).  
+Mailing Address:  
+The Apache Software Foundation  
+1000 N West Street, Suite 1200  
+Wilmington, DE 19801  
+U.S.A.
+
+## Personnel
+The Apache Foundation is led by a dedicated leadership team consisting of the Board of Directors, officers, corporate officers, and Project Management Committee (PMC) chairs who ensure the governance and vision of the foundation align with its core principles of open-source software development.
```
Comment (RichardLitt) [inline organizations/apachefoundation.md:19]: Sounds like it came from their website. Is there more we could say here?

Comment (reginankenchor) [reply organizations/apachefoundation.md:14]: > Sounds like it came from their website. Is there more we could say here?

Comment (reginankenchor) [reply organizations/apachefoundation.md:14]: > Sounds like it came from their website. Is there more we could say here?

Removed, thank you @RichardLitt

File: organizations/apachefoundation.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+## Funders
+The foundation receives support from a variety of sponsors and organizations dedicated to advancing open-source software. Funders include major tech companies like Google, Microsoft, Amazon Web Services, IBM, and many others that contribute to the sustainability and growth of Apache projects. ASF relies on these sponsors to maintain its projects and ensure their continued success.
+
+## Members
+Apache Software Foundation members are individuals who have demonstrated ongoing commitment to ASF by contributing to its projects. ASF members are crucial to the foundation's open-source initiatives, and their contributions have a lasting impact on the entire software ecosystem. A list of ASF members and contributors can be found [here](https://www.apache.org/foundation/members.html).
+
+### Academic Collaborations
+The Apache Foundation collaborates with various academic institutions worldwide to promote the use and development of open-source software. Notable collaborations include partnerships with:
+
+- **University of California, Berkeley**, known for contributions to projects like Apache Spark.
```
[the remark this answers is not in the document]
Comment (reginankenchor) [reply organizations/apachefoundation.md]: > Links to these unis?

@RichardLitt adjustment for universities we have on the list

**Comment by RichardLitt** [comment]
Merged! This is great. :)

**Pull request merged**